packages/test/service-load-test/src/nodeStressTest.ts

```diff
@@ -139,6 +142,10 @@ async function main() {
         process.exit(-1);
     }
 
+    if (driveId) {
+        config.driveId = driveId;
```

Comment:
Since you're adding the parameter and the default behavior, I don't think we also need the config file support for drive id. The default that's looked up will be sufficient most of the time, and when not just use the command parameter.

If that makes sense to you, please go ahead and remove driveId from the config object and just pass the value directly into `initialize`.  Thanks!